spec/lib/publishing_api_finder_publisher_spec.rb

```diff
@@ -19,7 +19,7 @@
       }
     end
 
-    def make_metadata base_path, overrides = {}
+    def make_metadata(base_path, overrides = {})
```

Comment:
Ah yeah. Meant to do that in a separate commit but forgot I'd changed it. Added it to an extra commit and pushed it.